app/Providers/RouteServiceProvider.php

```diff
@@ -2,98 +2,66 @@
 
 namespace App\Providers;
 
+use Illuminate\Cache\RateLimiting\Limit;
 use Illuminate\Foundation\Support\Providers\RouteServiceProvider as ServiceProvider;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\RateLimiter;
 use Illuminate\Support\Facades\Route;
 
 class RouteServiceProvider extends ServiceProvider
 {
 	/**
-	 * This namespace is applied to your controller routes.
+	 * The path to the "home" route for your application.
 	 *
-	 * In addition, it is set as the URL generator's root namespace.
+	 * This is used by Laravel authentication to redirect users after login.
 	 *
 	 * @var string
 	 */
-	protected $namespace = 'App\Http\Controllers';
-	protected $install_namespace = 'App\Http\Controllers\Install';
+	public const HOME = '/home';
```

Comment:
this is because of the change in `route.php`
Before you were setting the controllers by `'SomethingController@methodToCall'`
The syntax switches to `[SomethingController::class, 'methodToCall']`

Before we needed the namespace to prefix the `SomethingController` in order to access the class (and resolve it). Now we directly work with the Class name.